src/cnxml/deserialize.js

```diff
@@ -148,7 +148,7 @@ const mixed = type => (el, next) => ({
  * @see BLOCK
  */
 const BLOCK_TAGS = {
-    caption: text('figure_caption'),
+    caption: text('caption'),
```

Comment:
I'd prefer the default `<caption>` deserializer to check whether it's a child of `<figure>`, and skip it (return `undefined`) otherwise. This way we won't rely on a non-existent block (`'caption'`) and it should be easier to introduce a custom deserializer.

Reply:
When I'm doing this on the front:
```
function caption(el: Element, next: (nodes: any) => any) {
  if (el.parentElement && el.parentElement.tagName === 'TABLE') {
    return {
      type: 'table_caption',
      nodes: next(el.childNodes),
    }
  }
  return undefined
}
```
I'm getting `Error: `Block.fromJSON` requires a `type` string.`

I've tried with `return null`, `return undefined` and just `return`.